Adapter_oM/PriorityComparer.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
```

Comment:
Think adding a comparer for something like this is overkill - Think we can solve this in the Engine instead

Reply:
@IsakNaslundBh,
PriorityComparer removed as suggested.